doc/topics/releases/3005.rst

```diff
@@ -1,7 +1,135 @@
 .. _release-3005:
 
 =============================================
-Salt 3005 Release Notes - Codename Phosphorus
+Salt 3005 release notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+
+Python 3.5 and 3.6 Deprecation
+------------------------------
+
+This will be the last release we will support Python versions 3.5 and 3.6.
```

Comment:
**Suggestion (rephrase for clarity, blocking):** Instead of "This will be" let's say "The Phosphorus 3005 is the last release..."